deploy/helm/flightctl/templates/db/_db-migration-job.tpl

```diff
@@ -21,7 +21,7 @@ metadata:
   name: {{ $name }}
   namespace: {{ default $ctx.Release.Namespace $ctx.Values.global.internalNamespace }}
   labels:
-    app: flightctl-db-migration
+    flightctl.service: flightctl-db-migration
```

Comment:
There's a `spec.template.metadata.labels` later on this file that uses `app: flightctl-db-migration`, is that correct?